RELEASE_PROCESS.md

```diff
@@ -188,7 +191,7 @@ It's crucial to make the effort of thinking about what could happen in case a re
 ### Stable Release Managers
```

Comment:
we have to redo this entire doc, adding to my todo.

Reply:
Okay, I'll let you merge in case you want to push here.